aptos-move/aptos-aggregator/src/delayed_change.rs

```diff
@@ -220,20 +276,10 @@ mod test {
     #[test]
     fn test_merge_delta_into_delta() {
         let aggregator_change1: DelayedChange<DelayedFieldID> = Apply(AggregatorDelta {
-            delta: DeltaOp::new(SignedU128::Positive(10), 100, DeltaHistory {
-                max_achieved_positive_delta: 30,
-                min_achieved_negative_delta: 15,
-                min_overflow_positive_delta: Some(90),
-                max_underflow_negative_delta: Some(25),
-            }),
+            delta: DeltaWithMax::new(SignedU128::Positive(10), 100),
         });
         let aggregator_change2 = Apply(AggregatorDelta {
-            delta: DeltaOp::new(SignedU128::Positive(20), 100, DeltaHistory {
-                max_achieved_positive_delta: 25,
-                min_achieved_negative_delta: 20,
-                min_overflow_positive_delta: Some(95),
-                max_underflow_negative_delta: Some(45),
-            }),
+            delta: DeltaWithMax::new(SignedU128::Positive(20), 100),
```

Comment:
Why are we using DeltaWithMax for Aggregator V2? We need to track history for Aggregator V2 right?

Reply:
with V2 extension tracks DeltaWithMax only, and DeltaHistory is tracked in CapturedReads instead.

take a look at unit tests in CapturedReads - I've moved history tests from extension to there